README.md

```diff
@@ -10,33 +10,58 @@ Inmanta Documentation on [Read the Docs](https://inmanta.com/resources/docs/)
 
 ## Features
 
-For now, just basic Syntax Coloring
 
-![Screenshot](images/screenshot.png)
+### Syntax highlighting
+The extension makes your life easier by highlighting syntax keywords or the Inmanta language.
 
-## Alpha Features
+![Syntax coloring screenshot](images/screenshot-syntax-coloring.png)
 
 ### Code navigation
+When you hover an element while pressing `Ctrl`, an overview of its Entity type is shown. If you click it (while still having `Ctrl` pressed) you can navigate to the definition of the entity.
 
-To enable ctrl-click for code navigation, take the following steps:
+> :bulb: To work properly, this feature requires the Inmanta Language Server to be running:  
+> In the inmanta extension configuration (`ctrl+, > extensions > inmanta`), set `LS`: to Enabled
```

Comment:
shouldn't we enable by default?

Reply:
I guess it wasn't that useful before when more steps where required for the installation anyway but it could be now?

Reply:
I don't see any other open issue that would be relevant to include that change if required, so I will wait a bit more to merge this one.